munimap_digitize/munimap_digitize/static/js/app.js

```diff
@@ -6,6 +6,9 @@ proj4.defs('EPSG:25832', '+proj=utm +zone=32 +ellps=GRS80 +towgs84=0,0,0,0,0,0,0
 register(proj4);
 
 require('anol/src/anol/anol.js');
+require('angular-schema-form');
+require('angular-schema-form-bootstrap');
+
```

Comment:
Hopefully this is enough XD It was introduced here first, where it already was an early loading: https://github.com/stadt-bielefeld/bielefeldGEOCLIENT/commit/942a554224880b73bff05be2ed02495c18086cc1

Reply:
I can also put it above `require('anol/src/anol/anol.js')` if this makes things more robust.

Reply:
Done in https://github.com/stadt-bielefeld/bielefeldGEOCLIENT/pull/61/commits/5e2783780acc116f5ec6a58cc257dd7533b6aee2